lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -59,29 +62,45 @@ def supported_features(cls) -> List[Type[Feature]]:
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
         libvirt_events_thread.init()
 
+        self.qemu_platform_runbook = self.runbook.get_extended_runbook(
+            QemuPlatformSchema, type_name=QEMU
+        )
+        self.libvirt_conn_str = "qemu:///system"
+
+        if self.qemu_platform_runbook.is_host_remote():
+            self.host_node = RemoteNode(schema.Node(name="qemu-host"), 0, "qemu-host")
```

Comment:
You should still pre-declare these variables in the `__init__` function, so that type checking can work correctly. For example:
```python3
    def __init__(self, runbook: schema.Platform) -> None:
        super().__init__(runbook=runbook)
        self.host_node: Node
```